atd-vzv/src/views/map/InfoBox/MapInfoBox.js

```diff
@@ -3,73 +3,74 @@ import { Popup } from "react-map-gl";
 import InfoCard from "./InfoCard";
 import moment from "moment";
 import styled from "styled-components";
-import {
-  StyledDesktopInfo,
-  StyledMobileInfo,
-  setPopupPosition,
-} from "./infoBoxStyles";
+import { StyledMobileInfo, setPopupPosition } from "./infoBoxStyles";
 
-const MapInfoBox = ({
-  selectedFeature,
-  setSelectedFeature,
-  isTablet,
-  type, // id of feature layer
-}) => {
-  const popupInfo = selectedFeature && selectedFeature.properties;
-  const popupX = popupInfo.pixelCoordinates && popupInfo.pixelCoordinates.x;
+const MapInfoBox = React.memo(
+  ({
+    selectedFeature,
+    setSelectedFeature,
+    type, // id of feature layer
+  }) => {
```

Comment:
This is very cool! 👀  Out of curiosity, is there any particular reason we are using memo? Are we trying to improve on render performance?

Reply:
So I did this because the parent component is being rerendered because of the animated icon and I was not able to access the copy and close button of this child component. With React.memo, it essentially prevents this component from being rerendered. That's my understanding of how this works, but I could be wrong.

Reply:
@jgabitto that's smart! Glad you were able to resolve it with memo 📝 🙌